types/errors/errors.go

```diff
@@ -65,6 +65,12 @@ var (
 	// ErrNoSignatures to doc
 	ErrNoSignatures = Register(RootCodespace, 16, "no signatures supplied")
 
+	// ErrInvalidId to doc
+	ErrInvalidID = Register(RootCodespace, 17, "invalid identifier")
```

Comment:
I think we can add these in an `error.go` within `x/ibc/24-host/` to keep the error within the IBC context.

Reply:
Ok, should IBC have its own codespace?

Reply:
you can just pass the codespace as a param